tools/api-markdown-documenter/src/api-item-transforms/helpers/Helpers.ts

```diff
@@ -568,10 +567,8 @@ export function createDeprecationNoticeSection(
 				{ bold: true },
 			),
 			LineBreakNode.Singleton,
-			new SpanNode(transformTsdoc(deprecatedBlock, apiItem, config), {
-				italic: true,
```

Comment:
Formatting these contents as italic was a completely subjective choice. Additionally, the fluid-framework website build unconditionally overrides this default behavior. So, making this unformatted text affects no current users beyond our unit tests.